src/components/Search/SearchPageHeader.tsx

```diff
@@ -43,36 +57,64 @@ function SearchPageHeader({query, selectedItems = {}, hash, clearSelectedItems,
         finished: {icon: Illustrations.CheckmarkCircle, title: translate('common.finished')},
     };
 
-    const selectedItemsKeys = Object.keys(selectedItems ?? []);
+    const selectedTransactionsKeys = Object.keys(selectedTransactions ?? []);
 
     const headerButtonsOptions = useMemo(() => {
-        const options: Array<DropdownOption<SearchHeaderOptionValue>> = [];
-
-        if (selectedItemsKeys.length === 0) {
-            return options;
+        if (selectedTransactionsKeys.length === 0) {
+            return [];
         }
 
-        const itemsToDelete = Object.keys(selectedItems ?? {}).filter((id) => selectedItems[id].canDelete);
+        const options: Array<DropdownOption<SearchHeaderOptionValue>> = [
+            {
+                icon: Expensicons.Download,
+                text: translate('common.download'),
+                value: CONST.SEARCH.BULK_ACTION_TYPES.EXPORT,
+                shouldCloseModalOnSelect: true,
+                onSelected: () => {
+                    if (isOffline) {
+                        setOfflineModalOpen?.();
+                        return;
+                    }
+                    clearSelectedTransactions?.();
```

Comment:
We agreed on the doc to not clear the selection after downloading 

https://docs.google.com/document/d/1dwKo6o6PK-t5DlQlmZzP-B6uIvPOv74d6hc4rDgu8fs/edit?disco=AAABP0x5_zc


```suggestion
```

Reply:
Ok, I missed it